src/components/berths/Berth/BerthDetails/index.tsx

```diff
@@ -1,18 +1,26 @@
 import classNames from 'classnames';
 import React from 'react';
 import { FormattedMessage, injectIntl } from 'react-intl';
-import Icon from '../../../common/Icon';
+
+import Icon, { IconNames } from '../../../common/Icon';
+
 import './BerthDetails.scss';
-import BerthDetailsProps from './types';
 
-const BerthDetails = ({ iconName, available, value, titleId }: BerthDetailsProps) => {
+interface Props {
+  iconName?: IconNames;
+  available: boolean;
+  value?: number | null;
+  titleId: string;
+}
+
+const BerthDetails = ({ iconName, available, value, titleId }: Props) => {
   return (
     <div
       className={classNames('vene-berth__details', {
         'vene-berth__details-not-available': !available
       })}
     >
-      {value && <div className="vene-berth__details__value">{value}</div>}
+      {value !== undefined && <div className="vene-berth__details__value">{value || '-'}</div>}
```

Comment:
Isn't `value` is good enough, this check is not valid

Reply:
If the value is provided we should display it in any case, meaning, even if it comes as `null` from GraphQL we should render `-` instead.
We only omit this value when we don't provide it to the component.

Reply:
`value ? [Component] : '-'`

Reply:
This condition will fail when `value === null`